apps/src/code-studio/components/SoundCategory.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import * as color from "../../util/color";
+
+/**
+ * A component for displaying a sound category.
+ */
+var SoundCategory = React.createClass({
+  propTypes: {
+    displayName: React.PropTypes.string,
+    category: React.PropTypes.string,
+    onSelect: React.PropTypes.func.isRequired
+  },
+
+  selectCategory: function () {
+    this.props.onSelect(this.props.category);
+  },
+
+  render: function () {
+    var styles = {
+      category: {
+        backgroundColor: color.cyan,
+        border: 'solid 0px',
+        borderRadius: '5px',
+        width: '175px',
+        padding: '10px',
+        margin: '10px',
```

Comment:
Also generally prefer to use just number, i.e. `width: 175`. There are a few exceptions where that doesnt work, such as lineHeight